magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/LifecycleRegistry.kt

```diff
@@ -29,7 +29,7 @@ internal class LifecycleRegistry : LifecycleAware {
       }
     }
 
-  fun attachToLifecycle(
+  internal fun attachToLifecycle(
```

Comment:
I'm curious what the value is of having this class `public` when pretty much all of its methods are `internal`. Care to share your thoughts?

Reply:
Hah you're right, this is dumb